biz.aQute.bndlib/src/aQute/bnd/service/result/FunctionWithException.java

```diff
@@ -0,0 +1,74 @@
+package aQute.bnd.service.result;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.function.Function;
+import java.util.function.Supplier;
+
+import aQute.lib.exceptions.Exceptions;
```

Comment:
Why does this import Exceptions? I thought you were trying to disconnect from that package?

Reply:
yup, mistake